model/histogram/float_histogram.go

```diff
@@ -414,6 +433,10 @@ func (h *FloatHistogram) Compact(maxEmptyBuckets int) *FloatHistogram {
 // of observations, but NOT the sum of observations) is smaller in the receiving
 // histogram compared to the previous histogram. Otherwise, it returns false.
 //
+//	This method will shortcut to true if a CounterReset is detected, and shortcut
+//	to false if NotCounterReset is detected. Otherwise it will do the work to detect
+//	a reset.
```

Comment:
This shouldn't be indented. (I think it will render as a code block if it is indented.)